spec/models/custom_attribute_spec.rb

```diff
@@ -10,8 +10,4 @@
   it "returns the value type of DateTime custom attributes" do
     expect(time_custom_attribute.value_type).to eq(:datetime)
   end
-
-  it "returns the value type of Fixnum custom attributes" do
-    expect(int_custom_attribute.value_type).to eq(:fixnum)
-  end
```

Comment:
cc @alongoldboim can you review this commit?  It's not clear why we're using hardcoded ruby class names here.  Do you know if it's needed?